test/common/pdbs_exist.go

```diff
@@ -0,0 +1,49 @@
+package common
+
+import (
+	"testing"
+
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+var (
+	namespaces = []Namespace{
+		{
+			Name: "redhat-rhmi-rhsso",
+			PodDisruptionBudgetNames: []string{
+				"keycloak",
+			},
+		},
+		{
+			Name: "redhat-rhmi-user-sso",
+			PodDisruptionBudgetNames: []string{
+				"keycloak",
+			},
+		},
+		// {
+		// 	Name: "redhat-rhmi-3scale",
+		// 	PodDisruptionBudgetNames: []string{
+		// 		"apicast-production",
+		// 		"apicast-staging",
+		// 		"backend-cron",
+		// 		"backend-listener",
+		// 		"backend-worker",
+		// 		"system-app",
+		// 		"system-sidekiq",
+		// 		"zync",
+		// 		"zync-que",
+		// 	},
+		// },
+	}
+)
+
+func TestIntegreatlyPodDisruptionBudgetsExist(t *testing.T, ctx *TestingContext) {
+	for _, namespace := range namespaces {
+		for _, podDisruptionBudgetName := range namespace.PodDisruptionBudgetNames {
+			_, err := ctx.KubeClient.PolicyV1beta1().PodDisruptionBudgets(namespace.Name).Get(podDisruptionBudgetName, v1.GetOptions{})
+			if err != nil {
+				t.Fatalf("PodDisruptionBudget %s not found in namespace %s - Error: %s", podDisruptionBudgetName, namespace.Name, err)
```

Comment:
I suggest using `t.Errorf` so that the log will be printed and the test marked as failed but the execution can continue in order to check the remaining PDBs
```suggestion
				t.Errorf("PodDisruptionBudget %s not found in namespace %s - Error: %s", podDisruptionBudgetName, namespace.Name, err)
```

Reply:
excellent, thanks @b1zzu . Will make this change now